### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by utkuozdemir**
Bump dependencies.

### CODE REVIEW

File: Pkgfile [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
+  cni_version: v1.3.0
+  cni_sha256: f9871b9f6ccb51d2b264532e96521e44f926928f91434b56ce135c95becf2901
+  cni_sha512: 87e186b3cd64f66280f5b2293dcdd1fc22cb8f51a248124fb622adc48a893348419ba4c29c4769dede4d9e60f2e9fea5d4198f10badb4ecd20a1551e0b344e10
 
   # renovate: datasource=github-tags depName=containerd/containerd
-  containerd_version: v1.6.21
-  containerd_ref: 3dce8eb055cbb6872793272b4f20ed16117344f8
-  containerd_sha256: 9452e95455d03a00d78ae0587595d0c18555bae7912068269efa25a724efe713
-  containerd_sha512: a94ed8b8b9e153c9dc370228b659fcba1b03b3c47c8b5fcb001bb1b4b158ea048159f5d8d8c5294e7da3444edd76fbd903d7b7faa84e3382807a583d757325e1
+  containerd_version: v1.7.1
```
Comment (frezbo) [inline Pkgfile:14]: let's skip this, we decided to go directly to v2

File: Pkgfile [hunk 2]
```diff
@@ -173,9 +173,9 @@ vars:
   arm_trusted_firmware_sha512: 958cf8f9e258638e59d0fbd8b053fce7d8a9ea2fc922686c9d20ea16f79f55219ac18a12ab240c528ee98e49c2e0eef4c963fdb255cc14b92437a5b3cffc8640
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
-  uboot_version: 2023.01
-  uboot_sha256: 69423bad380f89a0916636e89e6dcbd2e4512d584308d922d1039d1e4331950f
-  uboot_sha512: 417a28267eb7875820d08fafc7316f164663609378637539e71648b0b9b7d28796b6c381717f31b0ab6472805fefd32628ef7d1b2e7b9f3c51c8ad122993f679
+  uboot_version: 2023.04
```
Comment (frezbo) [inline Pkgfile:176]: this need a lot of work in fixing patches :sweat_smile: , unless want to do it

File: Pkgfile (hunk 1 above)
Comment (utkuozdemir) [reply Pkgfile:14]: hm, is there a fixed date for its release?

File: Pkgfile (hunk 2 above)
Comment (utkuozdemir) [reply Pkgfile:176]: Oh I see, it looks like hell :D. Let's leave it for now.

**Review by frezbo — APPROVED** [review]

**Pull request merged**